app/models/concerns/invitable.rb

```diff
@@ -0,0 +1,11 @@
+module Invitable
+  extend ActiveSupport::Concern
+
+  included do
+    has_many :invitations, inverse_of: :task, foreign_key: :task_id
+  end
+
+  def invitation_accepted(invitation)
```

Comment:
Unused method argument - `invitation`. If it's necessary, use `_` or `_invitation` as an argument name to indicate that it won't be used. You can also write as `invitation_accepted(*)` if you want the method to accept any arguments but don't care about them.